src/js/queries.js

```diff
@@ -0,0 +1,12 @@
+import gql from "graphql-tag";
+
+export const UserByUIDQuery = gql`
```

Comment:
Why this specific query?

Reply:
`UserByUIDQuery` is literally the only query that can be shared as a whole.